packages/components/src/utils/log/warnThatDeprecatedPropIsUsed.ts

```diff
@@ -0,0 +1,10 @@
+import { getTagNameWithoutPrefix } from '../tag-name';
+
+// TODO: add missing unit test
+export const warnThatDeprecatedPropIsUsed = (host: HTMLElement, prop: string): void => {
+  console.warn(
+    `[Porsche Design System] ${prop} is deprecated for component "${getTagNameWithoutPrefix(
```

Comment:
Message could be abstracted and reused since this is identical with the one in `warnIfDeprecatedPropValueIsUsed`.

Reply:
fixed it